src/components/ErrorState/ErrorState.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import * as Styles from './styles';
+
+const ErrorState: React.FC = () => {
+  return (
+    <Styles.Container>
+        <Styles.ContentContainer>
+            <Styles.Refresh fontSize="large"></Styles.Refresh>
+            <Styles.TextDiv>
+                <Styles.Text>An unexpected error has occurred.</Styles.Text>
+                <Styles.Text>Please try again!</Styles.Text>
+            </Styles.TextDiv>
```

Comment:
Instead to create the component by yourselves, what do you think to use the **snackbar** component? since this component already have some animations expected. https://mui.com/material-ui/react-snackbar/

Reply:
To use the snackbar requires a user action.  What do you think about just using the alert component instead?  https://mui.com/material-ui/react-alert/

Reply:
go ahead :D